src/utils/user-list-helpers.js

```diff
@@ -0,0 +1,190 @@
+import {
```

Comment:
These functions were moved out of `session-core` and largely rewritten

Reply:
Think we should limit the scope of this PR to the conversion of users to webids. Can refactor in a separate PR.

Reply:
I've reworked this PR so it is only a backend update to users so they work properly with both webIds and Community Solid Server. Since we are still discussing the user registration flow, I can put that (and the UI updates) in a separate PR.